.github/workflows/build.yml

```diff
@@ -102,12 +102,17 @@ jobs:
 
       - name: Set vars
         id: setvars
-        run: make gh-docker-vars
+        run: |
+          read -d "\n" file version repo suffix <<< $(make gh-docker-vars) || true
```

Comment:
Why a simple `make gh-docker-vars >> $GITHUB_OUTPUT` is not sufficient?

Reply:
I don't know, I've tried this way firstly, but it doesn't work.

Reply:
Ugh, I've just tried it one more time, see https://github.com/AnnaShaleva/neo-go/actions/runs/3470731554/jobs/5799301496, and seems that it works. Maybe I've made some mistake the previous time.

Reply:
Fixed.